src/main/java/model/Cars.java

```diff
@@ -0,0 +1,62 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.stream.Collectors.toList;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        if (cars.isEmpty()) {
+            throw new IllegalArgumentException("레이싱 게임에 참가할 자동차를 1대 이상 입력해주세요");
+        }
+
+        this.cars = cars;
+    }
+
+    public Cars move() {
+        List<Car> records = new ArrayList<>();
+        for (Car car : cars) {
+            Car moved = car.move();
+            records.add(moved);
+        }
+
+        return new Cars(records);
+    }
+
+    public List<String> winnerNames() {
+        Position position = new Position(maxPosition());
+        return this.cars.stream()
+                .filter(car -> car.isSamePosition(position))
+                .map(Car::getName)
+                .collect(toList());
+    }
+
+    private int maxPosition() {
+        return this.cars.stream()
+                .max(Car::compareTo)
+                .map(Car::getPosition)
+                .orElse(0);
+    }
+
+    public List<Car> getCars() {
+        return Collections.unmodifiableList(cars);
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
```suggestion
    public boolean equals(Object o) {
```
자동완성 코드인 것 같아요 여기에도 o 보다는 other 같은 의미있는 변수로 적어주는 게 좋지 않을까요?

Reply:
파라미터명 설정하는 걸 찾지 못해서 
우선 renaming 기능으로 수정 했습니다😂